shared/templates/template_BASH_kernel_module_disabled

```diff
@@ -4,4 +4,9 @@
 # complexity = low
 # disruption = medium
 # THIS FILE IS GENERATED. DO NOT EDIT!
```

Comment:
Those reminders were put there in https://github.com/OpenSCAP/scap-security-guide/commit/ae24612b1569d8c7c65630db892505541b29f7e0. Reminder to users the remediation files are auto-generated.